packages/server/src/service/pdfForm/pdfForm.proto.ts

```diff
@@ -1,12 +1,11 @@
 import fs from 'fs'
 import { toSignatureBuffer } from './util'
 import { sampleStateInfo } from '../letter'
-import { VirginiaInfo } from '../../common'
 import { fillVirginia } from './virginia'
 
 
 const main = async () => {
-  const data = await sampleStateInfo('Virginia') as VirginiaInfo
+  const data = await sampleStateInfo('Virginia')
```

Comment:
This is great!  Thank you.  So I'd like to pass that data into `pdfForm` so that we don't have to choose the state in this file.  Could you do that?  Just throw an error if it returns undefined instead of a buffer (it's a proto / script file so it's fine to throw errors).